src/components/meetups/MeetupDetailsPage.tsx

```diff
@@ -27,22 +31,38 @@ const styles = StyleSheet.create({
 });
 
 export default function MeetupDetailsPage({ route }: MeetupDetailsPageProps) {
-  // TODO: Load in real data of the passed in meeting
+  // Create state for the meetup to be loaded
   const { meetupID } = route.params;
-  const meetupTime = "3:00 PM";
-  const meetupDuration = "25";
-  const meetupLocation = "Online";
-  const meetupParticipants = ["John Smith", "Bob Smith", "Alice Smith"];
+  const [meetup, setMeetup] = useState<Meetup>(BLANK_MEETUP);
+
+  // Retrieve account information
+  const { refreshToken } = useTokenState();
+
+  // Load the meetup details
+  useConstructor(async () => {
+    const [meetupDetails] = await getMeetupDetails(refreshToken, meetupID);
+    setMeetup(meetupDetails);
+  });
+
+  // TODO: Timestamp currently is just an integer, need to convert it to a time
+  const { id, timestamp, duration, location, user_list } = meetup; // eslint-disable-line camelcase
+  // Convert users dict to a list of accepted users
+  const acceptedUsers = [];
+  for (const [key, value] of Object.entries(user_list)) {
+    if (value === 1) {
+      acceptedUsers.push(key);
+    }
+  }
```

Comment:
maybe we should think about adding a refactor to the meeting card UI to betting illustrate this

Reply:
I agree, it should be a simple enough refactor to do